helpers/constants.js

```diff
@@ -38,12 +38,19 @@ const USERS_DEFAULT_IMAGE =
   'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png';
 
 const CHANNEL_TYPE = {
-  CHAT: 1,
-  GROUP: 2,
+  CHAT: 0,
+  GROUP: 1,
+  GROUP_LOCATION: 2,
   TOPIC: 3,
   ANONYMOUS: 4
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `CHANNEL_TYPE` enum has been updated with new values. However, the corresponding string representations for these new types are missing in the `CHANNEL_TYPE_STRING` object. This could lead to inconsistencies when trying to use the string representation of a channel type.

```diff
  const CHANNEL_TYPE_STRING = {
    CHAT: 'messaging',
    GROUP: 'group',
+   GROUP_LOCATION: 'group_location',
    TOPIC: 'topic',
+   ANONYMOUS: 'anonymous'
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->